crates/re_viewer/src/store_hub.rs

```diff
@@ -498,7 +509,14 @@ impl StoreBundle {
     pub fn purge_fraction_of_ram(&mut self, fraction_to_purge: f32) {
         re_tracing::profile_function!();
 
-        for store_db in self.store_dbs.values_mut() {
+        // TODO(jleibs) Calling `purge_fraction_of_ram` on Blueprints appears to
+        // cause partial corruption. Blueprints are aggressively garbage-collected
```

Comment:
This is worrying!? What kind of corruption? Have you looked into it more?

Reply:
Haven't had a chance to dig in deeply. It seems to put the blueprint in a state that violates some aspect of the purging logic.

Reply:
Well now that I've changed the implementation I can no longer repro the problem. Some bad edge-case interaction with the previous (overcomplicated) purging logic. Restoring this for now since once we have undo buffer this will be a sensible thing to gc.